target/linux/bcm63xx/base-files/etc/board.d/01_leds

```diff
@@ -91,6 +91,12 @@ sercomm,h500-s-lowi|\
 sercomm,h500-s-vfes)
 	ucidef_set_led_netdev "wan" "WAN" "green:internet" "eth0.2"
 	;;
+technicolor,tg582n)
+	ucidef_set_led_netdev "lan" "LAN" "green:ethernet" "eth0.1"
+	ucidef_set_led_netdev "wan" "WAN" "green:broadband" "eth0.2"
```

Comment:
How is this going to work for the default configuration if the switch isn't configured?
EDIT: is any of the 4 ethernet ports labelled as WAN? If this is not the case, then you should remove the WAN LED config.